src/test/java/org/apache/commons/text/ExtendedMessageFormatTest.java

```diff
@@ -426,10 +517,13 @@ public void testSetFormatsByArgumentIndex() {
 
         @Override
         public StringBuffer format(final Object obj, final StringBuffer toAppendTo, final FieldPosition pos) {
-            return toAppendTo.append(((String)obj).toLowerCase());
+            return toAppendTo.append(((String) obj).toLowerCase());
```

Comment:
Whitespace changes like this are _good_ IMO, as they improve code clarity. Well done! 👍